typedoc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://typedoc.org/schema.json",
+  "out": "./docs/reference",
+  "entryPoints": [
+    "./src/index.ts"
+  ],
+  "plugin": ["typedoc-plugin-markdown"],
```

Comment:
By default typedoc is html this makes it markdown so it COULD be used in vitepress